src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -47,6 +53,21 @@
     <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\AES\AesFactory.cs">
       <Link>CommonTest\AlgorithmImplementations\AES\AesFactory.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\DSA\DSAFactory.cs" Condition="'$(TargetGroup)'=='netstandard1.7'">
```

Comment:
Do you know if we have a policy regarding including new files in the build for netcore50 or netcoreapp1.0? i.e. do we keep them in (the newly build package will have 'extra' functionality outside of the contract) or do we exclude the files from compilation if feasible? I know for netcore50 it is common to include additional methods but ifdef the functionality out (at least for preexisting files) leaving stub methods.